src/design-pages/idoverride_redesign.rst

```diff
@@ -0,0 +1,400 @@
+ID-Override - Re-Design
+#######################
+
+Related Tickets
+***************
+
+* `Infopipe ListByCertificate does not return users if more than 1 override matches`_
+* `[RFE] Need an option for sss_override so that uidnumber can be added locally even if it is not available on LDAP server`_
+* `Unexpected interaction between overriding the primary group and the 'auto_private_groups = true' option`_
+* `RFE: Permanently Persistent Local (on-system) SSS Overrides both automating importation (such as after the SSS DB is cleared and SSSD restarted) and a standard location for the override export files`_
+* `IPA in trust. Client not seeing user with override set on AD user`_
+* `User information is being removed from SSSD local Override database`_
+
+.. _Infopipe ListByCertificate does not return users if more than 1 override matches: https://github.com/SSSD/sssd/issues/4456
+.. _[RFE] Need an option for sss_override so that uidnumber can be added locally even if it is not available on LDAP server: https://bugzilla.redhat.com/show_bug.cgi?id=1896728
+.. _Unexpected interaction between overriding the primary group and the 'auto_private_groups = true' option: https://bugzilla.redhat.com/show_bug.cgi?id=1921315
+.. _RFE\: Permanently Persistent Local (on-system) SSS Overrides both automating importation (such as after the SSS DB is cleared and SSSD restarted) and a standard location for the override export files: https://bugzilla.redhat.com/show_bug.cgi?id=1939527
+.. _IPA in trust. Client not seeing user with override set on AD user: https://bugzilla.redhat.com/show_bug.cgi?id=2010512
+.. _User information is being removed from SSSD local Override database: https://bugzilla.redhat.com/show_bug.cgi?id=1452883
+
+
+Problem statement
+*****************
+The original design for id-overrides is described in
+https://www.freeipa.org/page/V4/Migrating_existing_environments_to_Trust and
+all the features described there should of course work as expected after the
+redesign as well. Only the implementation should be updated with performance
+and reliability in mind.
+
+Initial the id-overrides were planned as a feature for the IPA provider. Some
```

Comment:
The initial? Initially? I don't know what's wrong there but I'd definitely change it